templates/outerDescribe.handlebars

```diff
@@ -1,5 +1,9 @@
 'use strict';
 var chai = require('chai');
+/*eslint-disable*/
+var ZSchema = require('z-schema');
```

Comment:
@mohsen1 the schema is passed when you call ```validator.validate(yourJSON, schema)```, the empty object you see is the options object